From: Jiri Kosina <jkosina@suse.cz>
Subject: Input: Add Acer Aspire 5710 to nomux blacklist
References: bnc#404881
Patch-mainline: submitted

Acer Aspire needs to be added to nomux blacklist, otherwise the touchpad
misbehaves.

Signed-off-by: Jiri Kosina <jkosina@suse.cz>

---
 drivers/input/serio/i8042-x86ia64io.h |    7 +++++++
 1 file changed, 7 insertions(+)

--- a/drivers/input/serio/i8042-x86ia64io.h
+++ b/drivers/input/serio/i8042-x86ia64io.h
@@ -348,6 +348,13 @@ static struct dmi_system_id __initdata i
 		},
 	},
 	{
+		.ident = "Acer Aspire 5710",
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Acer"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "Aspire 5710"),
+		},
+	},
+	{
 		.ident = "Gericom Bellagio",
 		.matches = {
 			DMI_MATCH(DMI_SYS_VENDOR, "Gericom"),
